docs/registry-images.md

```diff
@@ -72,16 +72,16 @@ Docker daemon and "inject" it into the  MicroK8s image cache like this:
 
 ```bash
 docker save mynginx > myimage.tar
-microk8s.ctr -n k8s.io image import myimage.tar
+microk8s.ctr --namespace k8s.io image import myimage.tar
```

Comment:
In the latest release we default to `k8s.io` for the namespace, this can be changed at `/var/snap/microk8s/current/args/ctr`.

Reply:
The right command here is without the "-n" argument.